templates/build-pod/template.json

```diff
@@ -0,0 +1,146 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "${NAME}",
+        "annotations": {
+            "openshift.io/display-name": "Maven Build Pod",
```

Comment:
Can we please get this `display-name` updated to be generic per the above convo?